integrationtest/orm/src/test/java/org/hibernate/search/v6poc/integrationtest/orm/OrmAnnotationMappingAccessTypeIT.java

```diff
@@ -36,6 +37,7 @@
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
+import junit.framework.AssertionFailedError;
```

Comment:
Maybe it's just in HV but we usually avoid the class from the `junit.`, wondering if `AssumptionViolationException` would work for you?

Reply:
Right, I'll change that to `Assert.fail`. No, `AssumptionViolationException` would not do, it's meant to ignore tests conditionally (e.g. for specific DB dialects), not to fail tests.